values.yaml

```diff
@@ -781,6 +790,21 @@ connectInject:
   # auth method for Connect inject, set this to the name of your auth method.
   overrideAuthMethodName: ""
 
+  # Custom configuration for the Kubernetes auth method to be used when
+  # global.acls.manageSystemACLs is set to true. Please see
+  # https://www.consul.io/docs/acl/auth-methods/kubernetes.html.
+  #
+  # This is useful when Consul servers are deployed outside of
+  # the Kubernetes cluster. Requires consul-k8s >= 0.14.0.
+  authMethodConfig:
+    # host corresponds to the Consul's Host configuration parameter
+    # ref: https://www.consul.io/docs/acl/auth-methods/kubernetes.html#host.
+    host: ""
+
+    # caCert is the base64-encoded PEM-encoded CA certificate string
+    # for this Kubernetes cluster.
+    caCert: ""
```

Comment:
Yeah, we can access it from the service account. I'm adding this config value in case you have a setup where the CA included in the service account would not be the same as the CA for the kube api host. 

You could provide only the host, I'll make it more clear in the comment for this option.

Reply:
How common is it for someone to have a different CA in the service account and the kube api host?

Reply:
Thinking about this a bit more, I don't think it makes too much sense to provide a custom CA. I'd like to punt on this and wait for someone to ask to make it configurable and, for the time being, only allow the host to be configurable.